source/includes/_pis.md

```diff
@@ -1,22 +1,95 @@
-# PISP Access
+# Payment Initiation Services API
 
-Access for companies authorised as [Payment Initiation Service Providers (AISPs)](https://www.fca.org.uk/account-information-service-ais-payment-initiation-service-pis) under PSD2.
+The Payment Initiation Services API lets authorised Payment Initiation Service Providers make outbound payments from 
+the accounts of our customers in the United Kingdom. All payments initiated through our Payment Initiation Services 
+API are sent through Faster Payments.
 
-Monzo provide Payments Initiation to firms authorised as Payment Initiation Service Providers. To request access, email `openbanking@monzo.com`.
+## Getting Access
 
-Access to our PISP sandbox is available on request and onboarding takes 1 month.
+The Open Banking team at Monzo manage access to the Account Information Services API. To get in touch, email us at 
+openbanking@monzo.com.
```

Comment:
This should be a mailto link too: `[openbanking@monzo.com](mailto:openbanking@monzo.com)`